### PR TITLE
Add fastboot tests to help avoid issues like #178

### PR DESCRIPTION
**Pull request by SergeAstapov**
This is built on top of #183.

Split into two separate PRs as I'm puzzled why this fails in CI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +56,8 @@
     "ember-cli": "~3.11.0",
     "ember-cli-dependency-checker": "^3.1.0",
     "ember-cli-eslint": "^5.1.0",
+    "ember-cli-fastboot": "^2.2.3",
+    "ember-cli-fastboot-testing": "^0.3.0",
```
Comment (jherdman) [inline package.json:60]: Perhaps try bumping to 0.4?

**Comment by SergeAstapov** [comment]
@jherdman finally got it working! `useYarn: true` option in `config/ember-try.js` made a difference

**Comment by jherdman** [comment]
🦄 ❤️ many thanks! ❤️ 🌈

**Comment by SergeAstapov** [comment]
@jherdman @ivanvotti could you please release 3.8.2 patch with this fix?

**Comment by jherdman** [comment]
Sure thing.

**Pull request merged**